### ISSUE TITLE
Marking Pro recipes

### ISSUE THREAD

**Issue by tbuerki** [everycook/EveryCook#174]
Recipes made by professionals (= same users that also have a public profile) should be highlighted in the search results as shown in the mock-up: the user picture appears to the right of the "don't like it" button, the background color of the recipe (< div class="resultArea" >) changes to a light grey and a bar with the caption "Pro" is shown across the top of the recipe image.

**Comment by swerder** [comment]
marking and show profile image done.
Problem is: the "user Name" is in the private DB, and selecting over both is not possible. (currently it show "TODO" as name)

currently the recipe is marked from Pro, if he change the recipe.
how should these be in future? (if pro user create a recipe? if its marked from admin as one?)
locking change, if a recipe marked as from pro? (except its the pro user or a admin)

**Comment by swerder** [comment]
name will now show.
copy visible professional profile from priv to normal db.

**Comment by tbuerki** [comment]
Photos and grey background work perfectly. Three more things on Pro recipes:
- At the top of the recipe image, an orange bar saying "PRO" should be shown (see mock-up)
- All Pro recipes should be shown first on the results page, only after that the search results with the non-pro recipes are shown
- The filters on the left side should include a filter called "Author" with checkboxes to select "Pro" and "Regular"

**Comment by swerder** [comment]
The PRO label is there already but the text was missing, now added.

**Comment by swerder** [comment]
search sort changed.

I do currently refactor / change the whole search logic (add "solr" search engine).
After that the filters would work better and I also add filter for Author.

(I moved the questions from above to new issue, assigned to you)
